components/builder-scheduler/src/data_store.rs

```diff
@@ -262,6 +261,43 @@ impl DataStore {
                            r#"CREATE OR REPLACE FUNCTION set_project_state_ident_v1 (pid bigint, jid bigint, state text, ident text) RETURNS void AS $$
                                   UPDATE projects SET project_state=state, job_id=jid, project_ident=ident, updated_at=now() WHERE id=pid;
                            $$ LANGUAGE SQL VOLATILE"#)?;
+
+        // Change the primary key for the groups table
+        migrator.migrate(
+            "scheduler",
+            r#"ALTER TABLE IF EXISTS groups DROP CONSTRAINT IF EXISTS groups_pkey;"#,
+        )?;
+        migrator.migrate(
+            "scheduler",
+            r#"CREATE SEQUENCE IF NOT EXISTS groups_id_seq;"#,
+        )?;
+        migrator.migrate("scheduler", r#"ALTER TABLE IF EXISTS groups ALTER COLUMN id SET DEFAULT next_id_v1('groups_id_seq');"#)?;
+        migrator.migrate(
+            "scheduler",
+            r#"ALTER TABLE IF EXISTS groups ADD PRIMARY KEY (id);"#,
+        )?;
+        migrator.migrate("scheduler",
+                     r#"CREATE OR REPLACE FUNCTION insert_group_v2 (
```

Comment:
It's possible I've misunderstood our versioning philosophy with these functions, but I thought part of the point of versioning them was that the old versions were allowed to continue to exist.  I've never removed a `_v1` function before, when adding a `_v2` function, nor have I seen that done in other places.  It's possible it _is_ happening though, and just escaping my notice.  What's the best way to get everyone on the same page?